en/releases/1.12.md

```diff
@@ -5,26 +5,65 @@
 
 ## Changes
 
+* [Common](#common)
+* [MAVLink](#mavlink)
+* [Commander](#commander)
 * [Multicoper](#multicopter)
+* [VTOL](#vtol)
+* [Control & Navigation](#control)
+
+### Common
+
+* **RTL Trigger based on remaining flight range ([PR#16399](https://github.com/PX4/PX4-Autopilot/pull/16399))**
+  * Calculates time to home, on RTL, taking into account vehicle speed, wind speed, and destination distance/direction
+* **Pre-emptive geofence breach ([PR#16400](https://github.com/PX4/PX4-Autopilot/pull/16400))**
+  * Triggers a breach if the _predicted_ current trajectory will result in a breach, allowing the vehicle to be re-routed to a safe hold position.
+
+### MAVLink
+
+* **MAVLink Ethernet configuration ([PR#14460](https://github.com/PX4/PX4-Autopilot/pull/14460))**
+  * MAVLink Ethernet channel settings such as UDP port, remote port and broadcast mode now can be changed dynamically via parameters.
+* **Support for querying `COMPONENT_INFORMATION` ([PR#16039](https://github.com/PX4/PX4-Autopilot/pull/16039))**
```

Comment:
FYI I updated this a little. The previous version just mentioned being able to get "supported commands". That's a great use case but we haven't implemented it. 
What we have so far is the infrastructure (nearly done) and the ability to get parameter metadata from the vehicle in a standardised way. 

The difference this adds is that currently QGC has to get parameter info from flight stacks in different formats. I'm not sure exactly how it gets to QGC but often the parameter info gets out of sync. In future QGC will get standardised info irrespective of the flight stack, which means that it can support parameters in other flight stacks much better. The info will always be "in sync", and translations will be able to be dynamically updated after release (in theory).

Generally, again FYI, the key benefit is that this can be used for any kind of metadata, and it is heirarchical. So we can use this without having to have new mavlink messages or transpose complicated nested structures into the flat structures that make sense for mavlink messages.

Reply:
I edited this to mention that this isn't just for GCS, any mavlink system that implements this can take advantage of it (EG: MAVSDK), but did leave a note that this was done with the intention to ease the burden of GCS's

Reply:
@mrpollo be aware that there are still breaking mavlink message changes coming for this, so it's not yet usable. I don't think it's worth mentioning until we have this working and enabled on both sides.

Reply:
@mrpollo I think @bkueng is right, as per usual :-). I have put this in a comment to hide it. We can unhide it and update if it managed to get in and somewhat stable in the final release: https://github.com/PX4/PX4-user_guide/pull/1142